es-app/src/views/gamelist/GridGameListView.cpp

```diff
@@ -71,7 +88,25 @@ GridGameListView::GridGameListView(Window* window, FileData* root) :
 	mDescription.setFont(Font::get(FONT_SIZE_SMALL));
 	mDescription.setSize(mDescContainer.getSize().x(), 0);
 	mDescContainer.addChild(&mDescription);
-
+	
+	mMarquee.setOrigin(0.5f, 0.5f);
+	mMarquee.setPosition(mSize.x() * 0.25f, mSize.y() * 0.10f);
+	mMarquee.setMaxSize(mSize.x() * (0.5f - 2*padding), mSize.y() * 0.18f);
+	mMarquee.setDefaultZIndex(35);
+	addChild(&mMarquee);
+
+	mImage.setOrigin(0.5f, 0.5f);
+	// Default to off the screen
+	mImage.setPosition(2.0f, 2.0f);
+	mImage.setMaxSize(1.0f, 1.0f);
+	mImage.setDefaultZIndex(10);
+	addChild(&mImage);
+
+	mVideo->setOrigin(0.5f, 0.5f);
+	mVideo->setPosition(mSize.x() * 0.25f, mSize.y() * 0.4f);
+	mVideo->setSize(mSize.x() * (0.5f - 2*padding), mSize.y() * 0.4f);
+	mVideo->setDefaultZIndex(15);
+	addChild(mVideo);
```

Comment:
Same as with the marquee, the video should be hidden by default.  This can be done with `mVideo->setVisible(false)`.

Reply:
The code remains the same, I prefer to set up default values, and finally the setVisible to false as suggested